### PR TITLE
Rk 621 comm metric fix

### PR DESCRIPTION
**Pull request by Robert-Krajcik**
Addresses issue #621

Looks like the `initialize_metric_table.sql` had already been updated.
The "Package Downloads" card is no longer flagged as non-metric

But the metric table in `upload_format.database` needs updating.

image

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -14,7 +14,8 @@
 * Identify non-riskmetric cards (#573)
 
 ### Squashed Bugs
-* Fixed busted button introduced with #547 (#592)
+* Fixed busted button introduced with #547 
+* Fixed tests showing "Package Downloads" community usage card as non-metric
```
Comment (AARON-CLARK) [inline NEWS.md:18]: You'll have to remove this from the v2.0.0 release. It should go up above, near the **development version** header.

File: NEWS.md [hunk 2]
```diff
@@ -14,7 +14,8 @@
 * Identify non-riskmetric cards (#573)
 
 ### Squashed Bugs
-* Fixed busted button introduced with #547 (#592)
+* Fixed busted button introduced with #547
```
Comment (AARON-CLARK) [inline NEWS.md:17]: I think we should keep the habit of tracking PRs in the news file. Could you leave the (# 592)?
```suggestion
* Fixed busted button introduced with #547 (#592)
```

**Review by AARON-CLARK — CHANGES_REQUESTED** [review]
Looks good Robert. I just made a few comments below. Could you also increment the dev version? Thanks!

File: NEWS.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
 # riskassessment (development version)
+* Fixed busted button introduced with #547 (#592)
```
Comment (AARON-CLARK) [inline NEWS.md:2]: Hi @Robert-Krajcik, this bullet point shouldn't be moved, correct? It belongs under "Squashed Bugs" for the `v2.0.0` release.

**Review by AARON-CLARK — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 3 above)
Comment (Robert-Krajcik) [reply NEWS.md]: OK, moved it back

**Review by AARON-CLARK — APPROVED** [review]
👍🏼 Looks great!

**Pull request merged**